frontend/src/pages/modelServing/screens/types.ts

```diff
@@ -31,20 +32,9 @@ export type ServingRuntimeToken = {
   editName?: string;
 };
 
-export type ServingRuntimeResources = {
-  limits: {
-    cpu: string;
-    memory: string;
-  };
-  requests: {
-    cpu: string;
-    memory: string;
-  };
-};
-
 export type ServingRuntimeSize = {
   name: string;
-  resources: ServingRuntimeResources;
+  resources: ContainerResources;
```

Comment:
We'll need to normalize on this -- I didn't go further than I needed for the type; #937 makes use of `ServingRuntimeResources`